containers/ftp/Dockerfile

```diff
@@ -3,15 +3,25 @@ FROM amazonlinux:latest
 LABEL org.prx.app="yes"
 
 RUN yum update -y \
-    && yum install -y python-pip tar wget xz \
-    && yum clean all
-RUN pip install boto3
+  && amazon-linux-extras enable ruby2.6 \
```

Comment:
Amazon already has a version of ruby ready for the `amazonlinux` image, this enables and installs it